packages/ui/form/fields.tsx

```diff
@@ -206,6 +206,8 @@ const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLForm
         ref={ref}
         onSubmit={(event) => {
           event.preventDefault();
+          event.stopPropagation();
```

Comment:
without this parent form( /event-types/[type]) handle submit function was executed before with incorrect previous data.